index.html

```diff
@@ -1201,15 +1201,17 @@ <h2>Core Vocabulary Definitions</h2>
           "td-context-ns-thing-mandatory">The <code>@context</code>
           name-value pair <em class="rfc2119" title=
           "MUST">MUST</em> contain the anyURI
-          <code>https://www.w3.org/2022/wot/td/v1.1</code> either
-          directly when of type <code>anyURI</code> or as first
-          element when of type <a href="#dfn-array" class=
-          "internalDFN" data-link-type="dfn">Array</a>.</span>
+          <code>https://www.w3.org/2022/wot/td/v1.1</code>
+          in order to identify the document as a TD 1.1 which would allow <a>Consumers</a> to use the newly introduced terms. When there are possibly TD 1.0 consumers the anyURI
+          <code>https://www.w3.org/2019/wot/td/v1</code>
+          <em class="rfc2119" title=
+          "MUST">MUST</em> be the first entry and the <code>https://www.w3.org/2022/wot/td/v1.1</code> <em class="rfc2119" title=
+          "MUST">MUST</em> be the second entry.</span>
           <span class="rfc2119-assertion" id=
           "td-context-ns-thing-optional">When <code>@context</code>
           is an <a href="#dfn-array" class="internalDFN"
           data-link-type="dfn">Array</a>, the anyURI
-          <code>https://www.w3.org/2022/wot/td/v1.1</code> <em class=
+          <code>https://www.w3.org/2019/wot/td/v1</code> <em class=
```

Comment:
I think this change was unintended, right?

Reply:
thanks. Should be fixed now